packages/renderer/vite.tests.setup.js

```diff
@@ -22,7 +22,7 @@ import 'vitest-canvas-mock';
 import typescript from 'typescript';
 import { expect } from 'vitest';
 
-global.window.matchMedia = () => {};
+global.window.matchMedia = vi.fn();
```

Comment:
question: can we have `beforeEach` in this file (or it's always considered as a global hook)

and move the definition of

```
vi.mocked(window.matchMedia).mockReturnValue({
    matches: false,
    addEventListener: vi.fn(),
    removeEventListener: vi.fn(),
  } as unknown as MediaQueryList);
```

here

so unit tests don't need to care of any `window.` methods except the one they really care